src/packages/excalidraw/README_NEXT.md

```diff
@@ -906,6 +906,17 @@ serializeAsJSON({
 
 Takes the scene elements and state and returns a JSON string. Deleted `elements`as well as most properties from `AppState` are removed from the resulting JSON. (see [`serializeAsJSON()`](https://github.com/excalidraw/excalidraw/blob/master/src/data/json.ts#L16) source for details).
 
+#### `serializeLibraryAsJSON`
+
+**_Signature_**
+
+<pre>
+serializeLibraryAsJSON({
+  libraryItems: <a href="https://github.com/excalidraw/excalidraw/blob/2fa69ddc329c327458d50ce6efde65d4e247b175/src/types.ts#L191">LibraryItems[]</a>,
```

Comment:
let's link to `master` branch. Will have potential to regress in terms of line position, but at least people won't be referring to old type as changes are made